src/storage/mod.rs

```diff
@@ -1249,6 +1327,11 @@ impl<E: Engine> Storage<E> {
         Ok(())
     }
 
+    /// Scan raw keys in [`start_key`, `end_key`), returns at most `limit` keys. If `end_key` is
```

Comment:
maybe need to explain the difference between a raw key and a mvcc key.

Reply:
I've added some description in `Storage`'s comments